ledger/participant-integration-api/src/main/resources/db/migration/oracle/V1__Init.sql

```diff
@@ -281,8 +281,8 @@ create index participant_events_event_sequential_id on participant_events (event
 create index participant_events_event_offset on participant_events (event_offset);
 
 -- TODO https://github.com/digital-asset/daml/issues/9493
--- create index participant_events_flat_event_witnesses_idx on participant_events (flat_event_witnesses);
--- create index participant_events_tree_event_witnesses_idx on participant_events (tree_event_witnesses);
+create index participant_events_flat_event_witnesses_idx on participant_events (JSON_ARRAY(flat_event_witnesses));
+create index participant_events_tree_event_witnesses_idx on participant_events (JSON_ARRAY(tree_event_witnesses));
```

Comment:
@danielporterda has confirmed this index is *NOT* being hit, presumably because it is indexing on JSON_ARRAY individual values but we are doing a JSON_EXISTS query which returns boolean

follow-on ticket https://github.com/digital-asset/daml/issues/9975 raised to investigate indexing of json array fields further if it is ascertained this is a performance bottleneck after full move to new appendonly schema